libmamba/src/core/query.cpp

```diff
@@ -575,7 +573,7 @@ namespace mamba
 
         void start_edge(node_id from, node_id to, const graph_type& g)
         {
-            m_is_last = g.get_edge_list(from).back() == to;
+            m_is_last = *(g.successors(from).rbegin()) == to;
```

Comment:
The `back` syntax is lighter, it might be convenient to add the method to the `vector_set` class.